test/signalrclient-e2e-tests/hub_connection_tests.cpp

```diff
@@ -181,4 +181,33 @@ TEST(hub_connection_tests, send_message_after_reconnect)
     ASSERT_FALSE(received_event->wait(200));
 
     ASSERT_EQ(*message, U("[\"Send: test\"]"));
-}
\ No newline at end of file
+}
+
+TEST(hub_connection_tests, progress_report)
```

Comment:
Add a test for the invoke overload that is invoking a server function returning void.